app/models/embed/purl/resource.rb

```diff
@@ -3,11 +3,17 @@
 module Embed
   class Purl
     class Resource
-      def initialize(resource, rights)
+      # @param [String] druid identifier without a namespace
+      # @param [Nokogiri::XML::Element] resource
+      # @param [Dor::RightsAuth] rights
+      def initialize(druid, resource, rights)
+        @druid = druid
         @resource = resource
         @rights = rights
       end
 
+      attr_reader :druid
+
```

Comment:
can we not pick the druid out of the resource XML? I'd assume it'd be in there somewhere

Reply:
It is not.